src/api/axiosService.ts

```diff
@@ -12,13 +13,21 @@ export interface HeaderType {
   'Content-Type'?: string;
 }
 
+export interface HeaderTypeEmail {
+  'X-Mail-Subject': EmailSubject;
+  'X-Mail-Sender-Address'?: string;
+  'X-Mail-Sender-Name'?: string;
```

Comment:
Are you still setting these two lines for possible inclusion in the Complete User Profile form, or are we going to go update keycloak first to update the name before making the API call?

Reply:
They won't be supported later unless we end up using them: Jason changed them on Thursday for our testing, but they will be limited later.